modules/statics/package.json

```diff
@@ -22,8 +22,7 @@
     "directory": "modules/statics"
   },
   "devDependencies": {
-    "nyc": "^15.0.0",
-    "should": "^13.2.3"
+    "nyc": "^15.0.0"
```

Comment:
nit: could remove nyc here

Reply:
There's quite a few unused packages throughout the repo. I'd like to address all of them at once in this ticket:
https://bitgoinc.atlassian.net/browse/BG-51022